addons/hr_expense/static/src/scss/hr_expense.scss

```diff
@@ -34,3 +34,7 @@
       height: 100%;
     }
 }
+
+td[name="description"] p {
```

Comment:
Maybe be a bit more restrictive and only do it for `p` following directly the `td`?

I imagine that we still want a normal margin for the other `p` if we use the HTML widget with some formatting and not only text

Reply:
Of course but can you help me understand a bit more? What do you mean by 

> the other `p` 

 I thought that having `td[name="description"]` would be restrictive enough as we'd only be targeting `p` tags inside `description` data cells only, so we'd be removing the margins for list views that have the `description` column.